gourmet/gtk_extras/LinkedTextView.py

```diff
@@ -169,17 +151,31 @@ def visibility_notify_event(self, text_view, event):
         self.set_cursor_if_appropriate (text_view, bx, by)
         return False
 
-    def follow_if_link (self, text_view, iter):
-        ''' Looks at all tags covering the position of iter in the text view,
+    def follow_if_link(self,
+                       text_view: 'LinkedTextView',
+                       itr: Gtk.TextIter) -> None:
+        """Looks at all tags covering the position of iter in the text view,
             and if one of them is a link, follow it by showing the page identified
             by the data attached to it.
-        '''
-        tags = iter.get_tags()
+        """
+        blue = Gdk.RGBA(red=0, green=0, blue=1., alpha=1.)
+
+        tags = itr.get_tags()
         for tag in tags:
-            href = tag.get_data('href')
-            if href:
-                self.emit('link-activated',href)
-                return True
+            color = tag.get_property('foreground-rgba')
+            if color and color == blue:
```

Comment:
why is there `if color`? Is the check `color == blue` not enough?

Reply:
Gtk objects can be compared only against other Gtk objects, and raise an `NotImplementedError`, see [the discussion here](https://gitlab.gnome.org/GNOME/pygobject/-/issues/414)

Reply:
okay. that makes sense. Thank you.